data/json/npcs/robofac/robofac_intercom/robofac_intercom_missions.json

```diff
@@ -145,6 +166,34 @@
       { "text": "Let's talk about something else.", "topic": "TALK_ROBOFAC_INTERCOM_SERVICES" }
     ]
   },
+  {
+    "id": [ "MISSION_ROBOFAC_INTERCOM_NRE_RECORDER_REPLACEMENT", "MISSION_ROBOFAC_INTERCOM_NRE_RECORDER_REPLACEMENT_Microphone" ],
+    "type": "talk_topic",
+    "dynamic_line": "Unfortunate, but not unfixable.  As before, we are willing to provide a replacement in exchange for an NRE microphone.",
+    "responses": [
+      {
+        "text": "Hook me up.  [Hand over the NRE microphone.]",
+        "condition": { "u_has_item": "xedra_microphone" },
+        "effect": [ { "u_consume_item": "xedra_microphone", "popup": true }, { "u_spawn_item": "nre_recorder" } ],
+        "topic": "MISSION_ROBOFAC_INTERCOM_NRE_RECORDER_REPLACEMENT_Complete"
+      },
+      {
+        "text": "What is an NRE microphone, and where can I find it?",
+        "topic": "MISSION_ROBOFAC_INTERCOM_NRE_RECORDER_REPLACEMENT_Microphone"
+      },
+      { "text": "On second thought, I'll get back to you on that.", "topic": "TALK_ROBOFAC_INTERCOM_SERVICES" }
+    ]
+  },
+  {
+    "id": "MISSION_ROBOFAC_INTERCOM_NRE_RECORDER_REPLACEMENT_Microphone",
+    "type": "talk_topic",
+    "dynamic_line": "The microphone is a static version of the same device used in the recorder, mounted on a tripod.  The glass ampoule is larger, but should look roughly the same in design.  They may be somewhat scarce given the current state of the world, but I would advise you to look inside of research labs.  Subway lines may be a good place to start."
+  },
+  {
+    "id": "MISSION_ROBOFAC_INTERCOM_NRE_RECORDER_REPLACEMENT_Complete",
+    "type": "talk_topic",
+    "dynamic_line": "&You fold up the microphone's legs, stuff it awkwardly into the tray, and slide it shut.  A few moments later, it slides open again, revealing a replacement NRE recorder.\n\n\"Try not to lose this one.\""
```

Comment:
```suggestion
    "dynamic_line": "You fold up the microphone's legs, stuff it awkwardly into the tray, and slide it shut.  A few moments later, it slides open again, revealing a replacement NRE recorder.\n\n\"Try not to lose this one.\""
```

Reply:
This is intentional - the & changes the syntax of the message so that it doesn't show the default quotations. You can see a screenshot of this in the OP. If I removed it, it'd display something like:

```
the intercom
"You fold up the microphone's legs..."
```

which is improper, since it's something you're doing, not something the intercom is saying.